dist/js/foundation.d.ts

```diff
@@ -333,21 +438,28 @@ declare module FoundationSites {
     destroy(): void;
   }
 
+  interface TooltipDefaults extends ITooltipOptions{
+
+  }
+
   interface ITooltipOptions {
+    disableForTouch?: boolean;
     hoverDelay?: number;
     fadeInDuration?: number;
     fadeOutDuration?: number;
-    disableHover?: boolean;
+    disableHover?: number;
```

Comment:
Documentation says this is a boolean. http://foundation.zurb.com/sites/docs/tooltip.html